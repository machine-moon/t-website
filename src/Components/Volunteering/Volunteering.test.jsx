import { test, expect } from "@jest/globals";
import { render } from "@testing-library/react";
import Volenteering from "./Volunteering";

test("renders Projects component correctly", () => {
  const { asFragment } = render(<Volenteering />);
  expect(asFragment()).toMatchSnapshot();
});
