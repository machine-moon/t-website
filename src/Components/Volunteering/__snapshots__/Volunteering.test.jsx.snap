// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`renders Projects component correctly 1`] = `
<DocumentFragment>
  <div
    class="volunteering"
    id="volunteering"
  >
    <div
      class="volunteering-title"
    >
      <h1>
        Volunteering Experience
      </h1>
    </div>
    <div
      class="volunteering-container"
    >
      <div
        class="project-card"
      >
        <img
          alt="TPU System Engineer"
          src="test-file-stub"
        />
        <div
          class="project-overlay"
        >
          <h2>
            TPU System Engineer
          </h2>
          <p>
            August 2024 - May 2025
          </p>
        </div>
      </div>
      <div
        class="project-card"
      >
        <img
          alt="VP of Technology & Projects"
          src="test-file-stub"
        />
        <div
          class="project-overlay"
        >
          <h2>
            VP of Technology & Projects
          </h2>
          <p>
            September 2024 - Present
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
